pkgs/top-level/aliases.nix

```
@@ -1534,6 +1534,7 @@ mapAliases ({
   truecrypt = throw "'truecrypt' has been renamed to/replaced by 'veracrypt'"; # Converted to throw 2022-02-22
   tuijam = throw "tuijam has been removed because Google Play Music was discontinued"; # Added 2021-03-07
   turbo-geth = throw "turbo-geth has been renamed to erigon"; # Added 2021-08-08
+  tvbrowser-bin = tvbrowser; # Added 2022-01-08
```

```suggestion
  tvbrowser-bin = throw "tvbrowser-bin was removed because it is now built from sources; use tvbrowse instead"; # Added 2022-01-08
```

After all, it makes no sense to call it `-bin`, even as an alias.